src/parse.d

```diff
@@ -1384,7 +1384,8 @@ final class Parser : Lexer
             *pudas = UserAttributeDeclaration.concat(*pudas, udas);
         }
         else
-            error("valid attributes are @property, @safe, @trusted, @system, @disable");
+            error("valid attributes are @%s, @%s, @%s, @%s, @%s, @%s",
```

Comment:
But what's the point of having Idgen if we're just hardcoding the identifiers anyway?